infra/experimental/chronos/cloudbuild.yaml

```diff
@@ -33,37 +37,68 @@ steps:
   - run
   - --entrypoint=/bin/sh
   - --env=SANITIZER=address
+  - --env=STORE_CCACHE=1
```

Comment:
I think we just need to update these right? and also set CACHE_DIR to /workspace/ccache